frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_day.dart

```diff
@@ -162,6 +162,9 @@ class CalendarDayCard extends StatelessWidget {
     });
 
     renderHook.addSelectOptionHook((selectedOptions, cardData, _) {
+      if (selectedOptions.isEmpty) {
+        return const SizedBox();
```

Comment:
```suggestion
        return const SizedBox.shrink();
```

Reply:
Hey @Xazin, I could be wrong this seems rather unnecessary since there is no child widget inside the `SizedBox`?

Reply:
You want it to take up the least amount if space possible. Exactly what the `.shrink()` constructor is made for.

Reply:
@richardshiue, I agree with @Xazin. As the flutter documentation said, using `Sizedbox.shrink` would be better/

> The simplest way to hide a child is just to not include it, or, if a child must be given (e.g. because the parent is a [StatelessWidget](https://api.flutter.dev/flutter/widgets/StatelessWidget-class.html)) then to use [SizedBox.shrink](https://api.flutter.dev/flutter/widgets/SizedBox/SizedBox.shrink.html) instead of the child that would otherwise be included.

Reply:
+1

Reply:
Got it thanks guys for the advice! I've update the PR